release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1663,6 +1742,12 @@ With this release, availability sets are no longer modified after creation, allo
 [id="ocp-release-note-storage-bug-fixes_{context}"]
 ==== Storage
 
+[discrete]
+[id="ocp-release-note-image-registry-bug-fixes_{context}"]
+==== Registry
```

Comment:
This entry isn't in alphabetical order

Reply:
I'll fix towards the end. Someone dropped this in here :(